<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="OriginalViewer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="visibilityViewer">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>441</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute Visibility</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Radius">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>99.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
